pkgs/top-level/aliases.nix

```diff
@@ -302,6 +302,7 @@ mapAliases ({
   msf = metasploit; # added 2018-04-25
   libmsgpack = msgpack; # added 2018-08-17
   mssys = ms-sys; # added 2015-12-13
+  mpv-with-scripts = throw "Use wrapMpv for editing the environment of mpv"; # added 20120-05-22
```

Comment:
Look at us, living in 2020, when @doronbehar is living in 20120 :P

Reply:
LOL. Fixed.